packages/preview-middleware-client/src/adp/quick-actions/enablement-validator.ts

```diff
@@ -0,0 +1,18 @@
+export interface EnablementValidatorSuccess {
```

Comment:
As this file only contains types, shouldn't the filename be `...-types` or the location of the file a `types` folder?

Reply:
There were also some functions here earlier, but they were later not needed and now only the types were left. I suppose we could more or rename, but I'm not sure how much value it brings. In general we do not split the types from the implementation here.